settings_dialog.js

```diff
@@ -79,14 +94,41 @@ function updateInstalledVersions(){
     });    
 }
 
+function moveInstalledFiles(filePath, destination){
+    
+    for (let i = 0; i < filePath.length; i++) {
+        let file = filePath[i];
+        let fileName = path.basename(filePath[i]);
+
+        movingFileModal.show();
```

Comment:
If this content doesn't change per loop iteration, it should be outside the loop. Also this way having it constantly pop up and close is not very good.